src/main.ts

```diff
@@ -39,41 +31,23 @@ async function bootstrap (): Promise<any> {
         ShutdownSignal.SIGSEGV,
         ShutdownSignal.SIGTERM,
     ]);
+    const bugsnagClient: BugsnagClient = app.get(BugsnagClient);
```

Comment:
I just looked up the type definition of `app.get(...)`:
_"Retrieves an instance of either injectable or controller, otherwise, throws exception."_

So I assume you have to try/catch it here. The same applies to the usage in user-new.

Reply:
I don't think so... what would I do in the `catch`, other than re-throw the error? We want the app to die on startup in such a case, so if we don't catch here, that's what we get. Plus, we get a pretty clear message on what went wrong:

`ERROR  : Nest could not find test element (this provider does not exist in the current context)`

Reply:
I just tested it. You are right, it really returns undefined if the bugsnag module is not imported and **does not throw**. I was just reading the documentation and it seems it is wrong... weird.
So please change the type definition here to: 
`const bugsnagClient: BugsnagClient | underfined = app.get(BugsnagClient);`

Reply:
So I looked a bit deeper into it.. why it's returning undefined here and not throwing is, because `GlobalLoggerModule` is importing `BugsnagModule`, so there it provides `undefined`, and so this resolves to undefined here, so the documentation is correct, just a bit complicated here.